### PR TITLE
Add Keyboard PH Arisu

### PR DESCRIPTION
**Pull request by devinceble**
Add new keyboard PH Arisu QMK support and VIA Compatibility

PH Arisu is base on https://github.com/FateNozomi/arisu-pcb but this used a Pro Micro
used for a local groupbuy in the philippines.

## Types of Changes

- [ ] Core
- [ ] Bugfix
- [ ] New feature
- [ ] Enhancement/optimization
- [X] Keyboard (addition or update)
- [X] Keymap/layout/userspace (addition or update)
- [ ] Documentation

## Issues Fixed or Closed by This PR

* 

## Checklist

- [x] My code follows the code style of this project.
- [ ] My change requires a change to the documentation.
- [ ] I have updated the documentation accordingly.
- [x] I have read the [**CONTRIBUTING** document](https://docs.qmk.fm/#/contributing).
- [ ] I have added tests to cover my changes.
- [x] I have tested the changes and verified that they work and don't break anything (as well as I can manage).

### CODE REVIEW

**Comment by akscanb** [comment]
Hey, dude I'm already in the process of trying to merge the arisu config into qmk. I also added via compatibility for it also.

**Comment by devinceble** [comment]
> Hey, dude I'm already in the process of trying to merge the arisu config into qmk. I also added via compatibility for it also.

Hey @akscanb this is different from the arisu though but we have same base project. That's why it PH Arisu not Arisu.

File: keyboards/ph_arisu/keymaps/via/keymap.c [hunk 1]
```diff
@@ -0,0 +1,54 @@
...
+		KC_TRNS, KC_TRNS, KC_TRNS, KC_TRNS, KC_TRNS, KC_TRNS, KC_TRNS,
+		KC_TRNS, KC_TRNS, KC_TRNS, KC_TRNS, KC_TRNS, KC_TRNS, KC_TRNS, KC_TRNS,
+		KC_TRNS, KC_TRNS, KC_TRNS, KC_TRNS, KC_TRNS, KC_TRNS,
+		KC_TRNS, KC_TRNS, KC_TRNS, KC_TRNS, KC_TRNS, KC_TRNS, KC_TRNS,
+		KC_TRNS, KC_TRNS, KC_TRNS, KC_TRNS, KC_TRNS, KC_TRNS, KC_TRNS,
+		KC_TRNS, KC_TRNS, KC_TRNS, KC_TRNS, KC_TRNS,
+		KC_TRNS, KC_TRNS, KC_TRNS, KC_TRNS)
+
+};
+
```
Comment (fauxpark) [inline keyboards/ph_arisu/keymaps/via/keymap.c:54]:
```suggestion
    LAYOUT(
        KC_ESC, KC_2, KC_4, KC_6, KC_8, KC_0, KC_EQL, KC_GRV,
        KC_1, KC_3, KC_5, KC_7, KC_9, KC_MINS, KC_BSLS,
        KC_TAB, KC_W, KC_R, KC_Y, KC_I, KC_P, KC_RBRC, KC_DEL,
        KC_Q, KC_E, KC_T, KC_U, KC_O, KC_LBRC, KC_BSPC,
        KC_CAPS, KC_S, KC_F, KC_H, KC_K, KC_SCLN, KC_ENT, KC_PGUP,
        KC_A, KC_D, KC_G, KC_J, KC_L, KC_QUOT,
        KC_LSFT, KC_X, KC_V, KC_N, KC_COMM, KC_SLSH, KC_PGDN,
        KC_Z, KC_C, KC_B, KC_M, KC_DOT, KC_LSFT, KC_UP,
        KC_LCTL, KC_LALT, KC_SPC, KC_LEFT, KC_RGHT,
        KC_SPC, KC_SPC, KC_LALT, KC_DOWN
    ),
    LAYOUT(
        RESET, _______, _______, _______, _______, _______, _______, _______,
        _______, _______, _______, _______, _______, _______, _______,
        _______, _______, _______, _______, _______, _______, _______, _______,
        _______, _______, _______, _______, _______, _______, _______,
        _______, _______, _______, _______, _______, _______, _______, _______,
        _______, _______, _______, _______, _______, _______,
        _______, _______, _______, _______, _______, _______, _______,
        _______, _______, _______, _______, _______, _______, _______,
        _______, _______, _______, _______, _______,
        _______, _______, _______, _______
    ),
    LAYOUT(
        _______, _______, _______, _______, _______, _______, _______, _______,
        _______, _______, _______, _______, _______, _______, _______,
        _______, _______, _______, _______, _______, _______, _______, _______,
        _______, _______, _______, _______, _______, _______, _______,
        _______, _______, _______, _______, _______, _______, _______, _______,
        _______, _______, _______, _______, _______, _______,
        _______, _______, _______, _______, _______, _______, _______,
        _______, _______, _______, _______, _______, _______, _______,
        _______, _______, _______, _______, _______,
        _______, _______, _______, _______
    ),
    LAYOUT(
        _______, _______, _______, _______, _______, _______, _______, _______,
        _______, _______, _______, _______, _______, _______, _______,
        _______, _______, _______, _______, _______, _______, _______, _______,
        _______, _______, _______, _______, _______, _______, _______,
        _______, _______, _______, _______, _______, _______, _______, _______,
        _______, _______, _______, _______, _______, _______,
        _______, _______, _______, _______, _______, _______, _______,
        _______, _______, _______, _______, _______, _______, _______,
        _______, _______, _______, _______, _______,
        _______, _______, _______, _______
    )
};
```

File: keyboards/ph_arisu/keymaps/via/keymap.c [hunk 2]
```diff
@@ -0,0 +1,54 @@
+#include "ph_arisu.h"
+
+const uint16_t PROGMEM keymaps[][MATRIX_ROWS][MATRIX_COLS] = {
+
```
Comment (fauxpark) [inline keyboards/ph_arisu/keymaps/via/keymap.c:4]:
```suggestion
```

File: keyboards/ph_arisu/keymaps/default/keymap.c [hunk 3]
```diff
@@ -0,0 +1,198 @@
...
+		KC_TRNS, KC_TRNS, KC_TRNS, KC_TRNS, KC_TRNS, KC_TRNS, KC_TRNS,
+		KC_TRNS, KC_TRNS, KC_TRNS, KC_TRNS, KC_TRNS, KC_TRNS, KC_TRNS, KC_TRNS,
+		KC_TRNS, KC_TRNS, KC_TRNS, KC_TRNS, KC_TRNS, KC_TRNS,
+		KC_TRNS, KC_TRNS, KC_TRNS, KC_TRNS, KC_TRNS, KC_TRNS, KC_TRNS,
+		KC_TRNS, KC_TRNS, KC_TRNS, KC_TRNS, KC_TRNS, KC_TRNS, KC_TRNS,
+		KC_TRNS, KC_TRNS, KC_TRNS, KC_TRNS, KC_TRNS,
+		KC_TRNS, KC_TRNS, KC_TRNS, KC_TRNS)
+
+};
+
```
Comment (fauxpark) [inline keyboards/ph_arisu/keymaps/default/keymap.c:198]:
```suggestion
    LAYOUT(
        KC_ESC, KC_2, KC_4, KC_6, KC_8, KC_0, KC_EQL, KC_GRV,
        KC_1, KC_3, KC_5, KC_7, KC_9, KC_MINS, KC_BSLS,
        KC_TAB, KC_W, KC_R, KC_Y, KC_I, KC_P, KC_RBRC, KC_DEL,
        KC_Q, KC_E, KC_T, KC_U, KC_O, KC_LBRC, KC_BSPC,
        KC_CAPS, KC_S, KC_F, KC_H, KC_K, KC_SCLN, KC_ENT, KC_PGUP,
        KC_A, KC_D, KC_G, KC_J, KC_L, KC_QUOT,
        KC_LSFT, KC_X, KC_V, KC_N, KC_COMM, KC_SLSH, KC_PGDN,
        KC_Z, KC_C, KC_B, KC_M, KC_DOT, KC_LSFT, KC_UP,
        KC_LCTL, KC_LALT, KC_SPC, KC_LEFT, KC_RGHT,
        KC_SPC, KC_SPC, KC_LALT, KC_DOWN
    ),
    LAYOUT(
        RESET, _______, _______, _______, _______, _______, _______, _______,
        _______, _______, _______, _______, _______, _______, _______,
        _______, _______, _______, _______, _______, _______, _______, _______,
        _______, _______, _______, _______, _______, _______, _______,
        _______, _______, _______, _______, _______, _______, _______, _______,
        _______, _______, _______, _______, _______, _______,
        _______, _______, _______, _______, _______, _______, _______,
        _______, _______, _______, _______, _______, _______, _______,
        _______, _______, _______, _______, _______,
        _______, _______, _______, _______
    )
};
```

File: keyboards/ph_arisu/keymaps/default/keymap.c [hunk 4]
```diff
@@ -0,0 +1,198 @@
+#include "ph_arisu.h"
```
Comment (fauxpark) [inline keyboards/ph_arisu/keymaps/default/keymap.c:1]:
```suggestion
#include QMK_KEYBOARD_H
```

File: keyboards/ph_arisu/keymaps/via/keymap.c [hunk 5]
```diff
@@ -0,0 +1,54 @@
+#include "ph_arisu.h"
```
Comment (fauxpark) [inline keyboards/ph_arisu/keymaps/via/keymap.c:1]:
```suggestion
#include QMK_KEYBOARD_H
```

File: keyboards/ph_arisu/rules.mk [hunk 6]
```diff
@@ -0,0 +1,26 @@
...
+BOOTMAGIC_ENABLE ?= yes	# Virtual DIP switch configuration(+1000)
+MOUSEKEY_ENABLE ?= yes	# Mouse keys(+4700)
+EXTRAKEY_ENABLE ?= yes	# Audio control and System control(+450)
+#CONSOLE_ENABLE ?= no	# Console for debug(+400)
+#COMMAND_ENABLE ?= no    # Commands for debug and configuration
+#SLEEP_LED_ENABLE ?= no  # Breathing sleep LED during USB suspend
+NKRO_ENABLE ?= yes		# USB Nkey Rollover - if this doesn't work, see here: https://github.com/tmk/tmk_keyboard/wiki/FAQ#nkro-doesnt-work
+#BACKLIGHT_ENABLE ?= no  # Enable keyboard backlight functionality
+#AUDIO_ENABLE ?= no
+#RGBLIGHT_ENABLE ?= no
```
Comment (fauxpark) [inline keyboards/ph_arisu/rules.mk:26]:
```suggestion
# Build Options
#   change yes to no to disable
#
BOOTMAGIC_ENABLE = yes      # Virtual DIP switch configuration
MOUSEKEY_ENABLE = yes       # Mouse keys
EXTRAKEY_ENABLE = yes       # Audio control and System control
CONSOLE_ENABLE = no         # Console for debug
COMMAND_ENABLE = no         # Commands for debug and configuration
# Do not enable SLEEP_LED_ENABLE. it uses the same timer as BACKLIGHT_ENABLE
SLEEP_LED_ENABLE = no       # Breathing sleep LED during USB suspend
# if this doesn't work, see here: https://github.com/tmk/tmk_keyboard/wiki/FAQ#nkro-doesnt-work
NKRO_ENABLE = yes           # USB Nkey Rollover
BACKLIGHT_ENABLE = no       # Enable keyboard backlight functionality
RGBLIGHT_ENABLE = no        # Enable keyboard RGB underglow
MIDI_ENABLE = no            # MIDI support
BLUETOOTH_ENABLE = no       # Enable Bluetooth with the Adafruit EZ-Key HID
AUDIO_ENABLE = no           # Audio output
FAUXCLICKY_ENABLE = no      # Use buzzer to emulate clicky switches
```

File: keyboards/ph_arisu/info.json [hunk 7]
```diff
@@ -0,0 +1,291 @@
+{
+    "keyboard_name": "PH Arisu",
+    "url": "https://github.com/FateNozomi/arisu-pcb",
+    "maintainer": "qmk",
+    "width": 19.75,
+    "height": 5.25,
+    "layouts": {
+        "LAYOUT_default": {
+            "layout":
```
Comment (fauxpark) [inline keyboards/ph_arisu/info.json:9]: This is not meant to contain the KLE layout. Please use https://qmk.fm/converter to convert the raw KLE JSON to the proper format.

Comment (devinceble) [reply keyboards/ph_arisu/info.json]: Many Thanks Updated info.json here https://github.com/qmk/qmk_firmware/pull/9612/commits/a38dea6148b5000ed2ffc9a34e14e87ffc69bebb

File: keyboards/ph_arisu/keymaps/default/keymap.c [hunk 8]
```diff
@@ -0,0 +1,28 @@
...
+        KC_TAB, KC_W, KC_R, KC_Y, KC_I, KC_P, KC_RBRC, KC_DEL,
+        KC_Q, KC_E, KC_T, KC_U, KC_O, KC_LBRC, KC_BSPC,
+        KC_CAPS, KC_S, KC_F, KC_H, KC_K, KC_SCLN, KC_ENT, KC_PGUP,
+        KC_A, KC_D, KC_G, KC_J, KC_L, KC_QUOT,
+        KC_LSFT, KC_X, KC_V, KC_N, KC_COMM, KC_SLSH, KC_PGDN,
+        KC_Z, KC_C, KC_B, KC_M, KC_DOT, KC_LSFT, KC_UP,
+        KC_LCTL, KC_LALT, KC_SPC, KC_LEFT, KC_RGHT,
+        KC_SPC, KC_SPC, KC_LALT, KC_DOWN
+    ),
+    [1] = LAYOUT(
```
Comment (Erovia) [inline keyboards/ph_arisu/keymaps/default/keymap.c:16]: I see no layer switching key, so how would someone reach this layer?

File: keyboards/ph_arisu/readme.md [hunk 9]
```diff
@@ -0,0 +1,14 @@
+# PH ARISU
```
Comment (Erovia) [inline keyboards/ph_arisu/readme.md:1]:
```suggestion
# PH ARISU

![PH_ARISU](<an image of the board or PCB>)
```

File: keyboards/ph_arisu/info.json [hunk 10]
```diff
@@ -0,0 +1,12 @@
...
...0}, {"label":"5,1", "x":5.85, "y":2.0}, {"label":"4,2", "x":6.85, "y":2.0}, {"label":"5,2", "x":7.85, "y":2.0}, {"label":"6,1", "x":5.3, "y":3.0}, {"label":"7,1", "x":6.3, "y":3.0}, {"label":"6,2", "x":7.3, "y":3.0}, {"label":"7,2", "x":8.3, "y":3.0}, {"label":"8,2", "x":6.6, "y":4.0, "w":2}, {"label":"9,2", "x":8.6, "y":4.0, "w":1.25}, {"label":"8,1", "x":5.05, "y":4.05, "w":1.5}, {"label":"1,3", "x":8.45, "y":3.6}, {"label":"0,4", "x":9.45, "y":3.6}, {"label":"1,4", "x":10.45, "y":3.6}, {"label":"0,5", "x":11.45, "y":3.6}, {"label":"2,3", "x":8.05, "y":4.6000000000000005}, {"label":"3,3", "x":9.05, "y":4.6000000000000005}, {"label":"2,4", "x":10.05, "y":4.6000000000000005}, {"label":"3,4", "x":11.05, "y":4.6000000000000005}, {"label":"4,3", "x":8.2, "y":5.6000000000000005}, {"label":"5,3", "x":9.2, "y":5.6000000000000005}, {"label":"4,4", "x":10.2, "y":5.6000000000000005}, {"label":"5,4", "x":11.2, "y":5.6000000000000005}, {"label":"6,3", "x":8.75, "y":6.6000000000000005}, {"label":"7,3", "x":9.75, "y":6.6000000000000005}, {"label":"6,4", "x":10.75, "y":6.6000000000000005}, {"label":"9,3", "x":7.75, "y":7.6000000000000005, "w":2.75}, {"label":"9,4", "x":10.55, "y":7.65, "w":1.5}]
```
Comment (Erovia) [inline keyboards/ph_arisu/info.json:9]: You used the KLE -> QMK Configurator converter, right?  
Configurator does not currently support rotated keys, that's why this json is full of rounding errors. Also it doesn't really render properly in Configurator. (You can check by pressing Ctrl+Shift+I on Configurator and loading the json.)

Check the Alice's info.json to see how this should be fixed.

Comment (Erovia) [reply keyboards/ph_arisu/info.json:9]: Just noticed that https://github.com/qmk/qmk_firmware/tree/master/keyboards/arisu also misses this info.json. Once you figure it out, you can probably contribute the more-or-less same json for both boards.

File: keyboards/ph_arisu/config.h [hunk 11]
```diff
@@ -0,0 +1,36 @@
...
+
+/* Mechanical locking support. Use KC_LCAP, KC_LNUM or KC_LSCR instead in keymap */
+#define LOCKING_SUPPORT_ENABLE
+
+/* Locking resynchronize hack */
+#define LOCKING_RESYNC_ENABLE
+
+/* prevent stuck modifiers */
+#define PREVENT_STUCK_MODIFIERS
+
```
Comment (zvecr) [inline keyboards/ph_arisu/config.h:36]:
```suggestion
```

File: keyboards/ph_arisu/keymaps/default/keymap.c (hunk 8 above)
Comment (devinceble) [reply keyboards/ph_arisu/keymaps/default/keymap.c:16]: Copy I will add Mod(1)

File: keyboards/ph_arisu/info.json (hunk 10 above)
Comment (devinceble) [reply keyboards/ph_arisu/info.json:9]: Noted Many thanks.

File: keyboards/ph_arisu/keymaps/default/keymap.c (hunk 8 above)
Comment (devinceble) [reply keyboards/ph_arisu/keymaps/default/keymap.c:16]: Add Layer switch key https://github.com/qmk/qmk_firmware/pull/9612/commits/12df44367e16874058c7119fd95a80b388e47929

File: keyboards/ph_arisu/info.json (hunk 10 above)
Comment (devinceble) [reply keyboards/ph_arisu/info.json]: Updated info.json https://github.com/qmk/qmk_firmware/pull/9612/commits/34a8366fb228030b679fbeb73cff37706133af27

File: keyboards/ph_arisu/readme.md [hunk 12]
```diff
@@ -0,0 +1,16 @@
...
+
+![PH_ARISU](https://www.dropbox.com/s/warl3bnuo49gxl1/PHArisu.jpg?raw=1)
```
Comment (fauxpark) [inline keyboards/ph_arisu/readme.md:3]: Please rehost this on Imgur.

Comment (devinceble) [reply keyboards/ph_arisu/readme.md]: Copy Updated Image Host to Imgur

Comment (devinceble) [reply keyboards/ph_arisu/readme.md]: https://github.com/qmk/qmk_firmware/pull/9612/commits/43812d6d0bb9b93703f3a657a327dbf221778338

File: keyboards/ph_arisu/readme.md [hunk 13]
```diff
@@ -0,0 +1,16 @@
+# PH ARISU
+
+![PH_ARISU](https://i.imgur.com/qr2Tn2X.jpg)
+
+PCB is base on https://github.com/FateNozomi/arisu-pcb
+and used for a local groupbuy in the Philippines.
+
+* Keyboard Maintainer: [Neil Brian Ramirez](https://github.com/NightlyBoards)
+* Hardware Supported: PH ARISU PCB
```
Comment (drashna) [inline keyboards/ph_arisu/readme.md:9]: This uses a Pro Micro, right?

Comment (devinceble) [reply keyboards/ph_arisu/readme.md:9]: atmega32u4 on Pro Micro yes.

Comment (devinceble) [reply keyboards/ph_arisu/readme.md]: Updated Readme https://github.com/qmk/qmk_firmware/pull/9612/commits/bb47f65e8b563d189bfd864137468ccba7a0fa1e

**Comment by devinceble** [comment]
Any changes to make this PR to merge?

File: keyboards/ph_arisu/config.h [hunk 14]
```diff
@@ -0,0 +1,32 @@
...
+#include "config_common.h"
+
+/* USB Device descriptor parameter */
+#define VENDOR_ID       0x6172
+#define PRODUCT_ID      0x6060
+#define DEVICE_VER      0x0001
+#define MANUFACTURER    PH
+#define PRODUCT         Arisu
+#define DESCRIPTION     Keyboard
```
Comment (noroadsleft) [inline keyboards/ph_arisu/config.h:11]: `DESCRIPTION` has been removed from QMK's core and no longer serves a purpose.

```suggestion
```

File: keyboards/ph_arisu/ph_arisu.h [hunk 15]
```diff
@@ -0,0 +1,27 @@
...
+	K000, K001, K002, K003, K004, K005, K006, K007, \
+	K010, K011, K012, K013, K014, K015, K016,       \
+	K020, K021, K022, K023, K024, K025, K026, K027, \
+	K030, K031, K032, K033, K034, K035, K036,       \
+	K040, K041, K042, K043, K044, K045, K046, K047, \
+	K050, K051, K052, K053, K054, K055,             \
+	K060, K061, K062, K063, K064, K065,       K067, \
+	K070, K071, K072, K073, K074, K075, K076,       \
+	K080, K081, K082,                   K086, K087, \
+	            K092, K093, K094,       K096  \
```
Comment (noroadsleft) [inline keyboards/ph_arisu/ph_arisu.h:15]: Arranges the keycodes according to the keyboard's physical layout.

```suggestion
	K000, K010, K001, K011, K002, K012, K003,       K013, K004, K014, K005, K015, K006, K016, K007, K027, \
	K020, K030, K021, K031, K022, K032,       K023, K033, K024, K034, K025, K035, K026, K036,       K047, \
	K040, K050, K041, K051, K042, K052,       K043, K053, K044, K054, K045, K055,       K046,       K067, \
	K060, K070, K061, K071, K062, K072,       K063, K073, K064, K074, K065, K075,             K076,       \
	K080,       K081,       K082, K092,       K093,       K094,                         K086, K096, K087  \
```

File: keyboards/ph_arisu/keymaps/default/keymap.c [hunk 16]
```diff
@@ -0,0 +1,28 @@
...
+        _______, _______, _______, _______, _______, _______, _______,
+        _______, _______, _______, _______, _______, _______, _______, _______,
+        _______, _______, _______, _______, _______, _______, _______,
+        _______, _______, _______, _______, _______, _______, _______, _______,
+        _______, _______, _______, _______, _______, _______,
+        _______, _______, _______, _______, _______, _______, _______,
+        _______, _______, _______, _______, _______, _______, _______,
+        _______, _______, _______, _______, _______,
+        _______, _______, _______, _______
+    )
```
Comment (noroadsleft) [inline keyboards/ph_arisu/keymaps/default/keymap.c:27]:
```suggestion
    [0] = LAYOUT(
        KC_ESC,  KC_1,    KC_2,    KC_3,    KC_4,    KC_5,    KC_6,             KC_7,    KC_8,    KC_9,    KC_0,    KC_MINS, KC_EQL,  KC_BSLS, KC_GRV,  KC_DEL,
        KC_TAB,  KC_Q,    KC_W,    KC_E,    KC_R,    KC_T,             KC_Y,    KC_U,    KC_I,    KC_O,    KC_P,    KC_LBRC, KC_RBRC, KC_BSPC,          KC_PGUP,
        KC_LCTL, KC_A,    KC_S,    KC_D,    KC_F,    KC_G,             KC_H,    KC_J,    KC_K,    KC_L,    KC_SCLN, KC_QUOT,          KC_ENT,           KC_PGDN,
        KC_LSFT, KC_Z,    KC_X,    KC_C,    KC_V,    KC_B,             KC_N,    KC_M,    KC_COMM, KC_DOT,  KC_SLSH, KC_LSFT,                   KC_UP,
        MO(1),            KC_LALT,          KC_SPC,  KC_SPC,           KC_SPC,           KC_LALT,                                     KC_LEFT, KC_DOWN, KC_RGHT
    ),
    [1] = LAYOUT(
        RESET,   _______, _______, _______, _______, _______, _______,          _______, _______, _______, _______, _______, _______, _______, _______, _______,
        _______, _______, _______, _______, _______, _______,          _______, _______, _______, _______, _______, _______, _______, _______,          _______,
        _______, _______, _______, _______, _______, _______,          _______, _______, _______, _______, _______, _______,          _______,          _______,
        _______, _______, _______, _______, _______, _______,          _______, _______, _______, _______, _______, _______,                   _______,
        _______,          _______,          _______, _______,          _______,          _______,                                     _______, _______, _______
    )
```

File: keyboards/ph_arisu/keymaps/via/keymap.c [hunk 17]
```diff
@@ -0,0 +1,56 @@
...
+        _______, _______, _______, _______, _______, _______, _______,
+        _______, _______, _______, _______, _______, _______, _______, _______,
+        _______, _______, _______, _______, _______, _______, _______,
+        _______, _______, _______, _______, _______, _______, _______, _______,
+        _______, _______, _______, _______, _______, _______,
+        _______, _______, _______, _______, _______, _______, _______,
+        _______, _______, _______, _______, _______, _______, _______,
+        _______, _______, _______, _______, _______,
+        _______, _______, _______, _______
+    )
```
Comment (noroadsleft) [inline keyboards/ph_arisu/keymaps/via/keymap.c:54]:
```suggestion
    [0] = LAYOUT(
        KC_ESC,  KC_1,    KC_2,    KC_3,    KC_4,    KC_5,    KC_6,             KC_7,    KC_8,    KC_9,    KC_0,    KC_MINS, KC_EQL,  KC_BSLS, KC_GRV,  KC_DEL,
        KC_TAB,  KC_Q,    KC_W,    KC_E,    KC_R,    KC_T,             KC_Y,    KC_U,    KC_I,    KC_O,    KC_P,    KC_LBRC, KC_RBRC, KC_BSPC,          KC_PGUP,
        KC_LCTL, KC_A,    KC_S,    KC_D,    KC_F,    KC_G,             KC_H,    KC_J,    KC_K,    KC_L,    KC_SCLN, KC_QUOT,          KC_ENT,           KC_PGDN,
        KC_LSFT, KC_Z,    KC_X,    KC_C,    KC_V,    KC_B,             KC_N,    KC_M,    KC_COMM, KC_DOT,  KC_SLSH, KC_LSFT,                   KC_UP,
        MO(1),            KC_LALT,          KC_SPC,  KC_SPC,           KC_SPC,           KC_LALT,                                     KC_LEFT, KC_DOWN, KC_RGHT
    ),

    [1] = LAYOUT(
        RESET,   _______, _______, _______, _______, _______, _______,          _______, _______, _______, _______, _______, _______, _______, _______, _______,
        _______, _______, _______, _______, _______, _______,          _______, _______, _______, _______, _______, _______, _______, _______,          _______,
        _______, _______, _______, _______, _______, _______,          _______, _______, _______, _______, _______, _______,          _______,          _______,
        _______, _______, _______, _______, _______, _______,          _______, _______, _______, _______, _______, _______,                   _______,
        _______,          _______,          _______, _______,          _______,          _______,                                     _______, _______, _______
    ),

    [2] = LAYOUT(
        _______, _______, _______, _______, _______, _______, _______,          _______, _______, _______, _______, _______, _______, _______, _______, _______,
        _______, _______, _______, _______, _______, _______,          _______, _______, _______, _______, _______, _______, _______, _______,          _______,
        _______, _______, _______, _______, _______, _______,          _______, _______, _______, _______, _______, _______,          _______,          _______,
        _______, _______, _______, _______, _______, _______,          _______, _______, _______, _______, _______, _______,                   _______,
        _______,          _______,          _______, _______,          _______,          _______,                                     _______, _______, _______
    ),

    [3] = LAYOUT(
        _______, _______, _______, _______, _______, _______, _______,          _______, _______, _______, _______, _______, _______, _______, _______, _______,
        _______, _______, _______, _______, _______, _______,          _______, _______, _______, _______, _______, _______, _______, _______,          _______,
        _______, _______, _______, _______, _______, _______,          _______, _______, _______, _______, _______, _______,          _______,          _______,
        _______, _______, _______, _______, _______, _______,          _______, _______, _______, _______, _______, _______,                   _______,
        _______,          _______,          _______, _______,          _______,          _______,                                     _______, _______, _______
    )
```

File: keyboards/ph_arisu/readme.md [hunk 18]
```diff
@@ -0,0 +1,16 @@
+# PH ARISU
+
+![PH_ARISU](https://i.imgur.com/qr2Tn2X.jpg)
```
Comment (noroadsleft) [inline keyboards/ph_arisu/readme.md:3]: Swaps this image for a smaller version.

```suggestion
![PH_ARISU](https://i.imgur.com/qr2Tn2Xl.jpg)
```

File: keyboards/ph_arisu/readme.md [hunk 19]
```diff
@@ -0,0 +1,16 @@
...
+
+PCB is base on https://github.com/FateNozomi/arisu-pcb
+and used for a local groupbuy in the Philippines.
+
+* Keyboard Maintainer: [Neil Brian Ramirez](https://github.com/NightlyBoards)
```
Comment (noroadsleft) [inline keyboards/ph_arisu/readme.md:8]: Did NightlyBoards do the PCB modification (from the FateNozomi Arisu)? Usually, this line would refer to your account.

File: keyboards/ph_arisu/rules.mk [hunk 20]
```diff
@@ -0,0 +1,31 @@
...
+MCU = atmega32u4
+
+# Bootloader selection
+#   Teensy       halfkay
+#   Pro Micro    caterina
+#   Atmel DFU    atmel-dfu
+#   LUFA DFU     lufa-dfu
+#   QMK DFU      qmk-dfu
+#   ATmega32A    bootloadHID
+#   ATmega328P   USBasp
```
Comment (noroadsleft) [inline keyboards/ph_arisu/rules.mk:11]:
```suggestion
```

File: keyboards/ph_arisu/readme.md (hunk 19 above)
Comment (devinceble) [reply keyboards/ph_arisu/readme.md:8]: Yes is the one modified the PCB I just do the QMK and VIA for him.

File: keyboards/nightly_boards/ph_arisu/rules.mk [hunk 21]
```diff
@@ -0,0 +1,24 @@
...
+# Do not enable SLEEP_LED_ENABLE. it uses the same timer as BACKLIGHT_ENABLE
+SLEEP_LED_ENABLE = no       # Breathing sleep LED during USB suspend
+# if this doesn't work, see here: https://github.com/tmk/tmk_keyboard/wiki/FAQ#nkro-doesnt-work
+NKRO_ENABLE = yes           # USB Nkey Rollover
+BACKLIGHT_ENABLE = no       # Enable keyboard backlight functionality
+RGBLIGHT_ENABLE = no        # Enable keyboard RGB underglow
+MIDI_ENABLE = no            # MIDI support
+BLUETOOTH_ENABLE = no       # Enable Bluetooth
+AUDIO_ENABLE = no           # Audio output
+FAUXCLICKY_ENABLE = no      # Use buzzer to emulate clicky switches
```
Comment (fauxpark) [inline keyboards/nightly_boards/ph_arisu/rules.mk:24]:
```suggestion
BLUETOOTH_ENABLE = no       # Enable Bluetooth
AUDIO_ENABLE = no           # Audio output
```

**Comment by noroadsleft** [comment]
Thanks!

**Pull request merged**